proseco/catalog.py

```diff
@@ -439,7 +441,10 @@ def merge_cats(fids=None, guides=None, acqs=None):
         fids['p_acq'] = 0
         fids['sz'] = '8x8'
 
-    guide_size = '8x8' if len(fids) == 0 else '6x6'
+    if 'star_img_size' in kwargs and kwargs['star_img_size'] is not None:
```

Comment:
This is the way my brain is working today but this could probably be trimmed down.